.github/workflows/ci.yml

```
@@ -80,7 +80,8 @@ jobs:
         shell: bash -l {0}
         run: |
           python -m pip install nbmake
-          python -m pip install ".[dev,docs,test]"
+          python -m pip install ".[dev,docs,test,hpopt]"
+          python -m pip install optuna
```

```suggestion
          python -m pip install ".[dev,docs,test]"
          python -m pip install ".[hpopt]" optuna
```

This could fix the CI issue